src/util/types.ts

```diff
@@ -27,9 +31,31 @@ export type PresentationAttributesWithProps<P, T> = AriaAttributes &
   DOMAttributesWithProps<P, T> &
   SVGProps<T>;
 
-export type SymbolType = 'circle' | 'cross' | 'diamond' | 'square' | 'star' | 'triangle' | 'wye'
+export type SymbolType = 'circle' | 'cross' | 'diamond' | 'square' | 'star' | 'triangle' | 'wye';
 export type LegendType = 'plainline' | 'line' | 'square' | 'rect' | 'circle' | 'cross' |
-  'diamond' | 'star' | 'triangle' | 'wye' | 'none'
+  'diamond' | 'star' | 'triangle' | 'wye' | 'none';
+export type TooltipType = 'none';
+export interface Coordinate {
+  x: number;
+  y: number;
+};
+export enum ScaleType {
+  AUTO = 'auto',
+  LINEAR = 'linear', 
+  POW = 'pow', 
+  SQRT = 'sqrt', 
+  LOG = 'log',
+  IDENTITY = 'identity', 
+  TIME = 'time', 
+  BAND = 'band', 
+  POINT = 'point',
+  ORDINAL = 'ordinal', 
+  QUANTILE = 'quantile', 
+  QUANTIZE = 'quantize', 
+  UTC = 'utc', 
+  SEQUENTIAL = 'sequential', 
+  THRESHOLD = 'threshold',
+};
```

Comment:
```ts
export type ScaleType = 'auto' | 'linear' | 'pow' | ...
```
Isn't above is shorter and better than enum type? Will we use it like `scaleType == ScaleType.AUTO`?

Reply:
Good suggestion! I'll change this.